app/templates/signup.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div>
+    <style>
+    h2{
+        font-size: 8px;
+        color: grey;
+        }
+    </style>
+    <h1> Register </h1>
+    <form action="" method="post" novalidate>
+        {{form.hidden_tag() }}
+        
+        <p>
+            {{ form.name.label }}<br>
+            {{ form.name(size = 32)}}
+        </p>

```

Comment:
I would remove this

Reply:
That is probably best considering we already have a username.